### DISCUSSION TITLE
Consolidating issues in new milestone

### DISCUSSION THREAD

**Discussion by ludofischer** [cssnano/cssnano#1012] [General]
I think [4.1](https://github.com/cssnano/cssnano/milestone/12),  [4.2 ](https://github.com/cssnano/cssnano/milestone/13) and [4.3](https://github.com/cssnano/cssnano/milestone/14) are unlikely to be released. I'd like to close those milestones and move everrything in there aither to 5.0.0 or backlog. Any thoughts?

**Comment by alexander-akait** [comment]
Yes, let's do it

**Comment by anikethsaha** [comment]
+1 for this

**Comment by ludofischer** [comment]
Done! And I've  found an extra issue that will be fixed in the 5.0 release: https://github.com/cssnano/cssnano/issues/535 (was in the 4.1 milestone)
